molecule/default/molecule.yml

```diff
@@ -0,0 +1,37 @@
+---
+dependency:
+  name: galaxy
+  options:
+    role-file: molecule/default/requirements.yml
+driver:
+  name: podman
+platforms:
+  - name: collectd-test
+    image: docker.io/tripleomaster/centos-binary-collectd:current-tripleo-rdo
+provisioner:
+  name: ansible
+  log: true
+  playbooks:
+    prepare: ../common/prepare.yml
+  inventory:
+    host_vars:
+      collectd-test:
+        collectd_conf_output_dir: "/etc/collectd.d/"
+verifier:
+  name: ansible
+
+scenario:
+  name: default
+  test_sequence:
+    - lint
+    - destroy
+    - dependency
+    # - syntax
+    - create
+    - prepare
+    - converge
+    # - idempotence
```

Comment:
Why is this one commented out?

Reply:
The config dir is wiped before creating the configs, so that there aren't any extra plugins left that weren't in the plugin list.